framework/src/controller/helpers/configurator/index.js

```diff
@@ -0,0 +1,21 @@
+const yargs = require('yargs');
+const chainModule = require('../../../modules/chain');
+const APIModule = require('../../../modules/http_api');
+const Index = require('./configurator');
```

Comment:
shouldn't the variable name to be `configurator`? `Index` is not relevant though.

Reply:
Its something comes from IDE with refactor/rename. I will rename it.